pkg/utils/utils.go

```diff
@@ -183,3 +183,49 @@ func UploadFile(url, filePath string, formFields map[string]any) error {
 
 	return nil
 }
+
+func UploadFileToAzureBlob(url, filePath, contentFormat string) error {
+	fileInfo, err := os.Stat(filePath)
+	if err != nil {
+		return err
+	}
+
+	if fileInfo.Size() > maxFileSize {
+		return fmt.Errorf("file size %d exceeds the 1GB limit", fileInfo.Size())
```

Comment:
There is a lot of duplication between this function and `UploadFile()` above, and it's clear that you just copied and pasted it. Can we de-duplicate these functions in any way or at the very least define this shared string as a constant so it only needs to be updated in one place if we decide to change the file size limit in the future?

Reply:
The header and body input are all different for these two funcs. It is hard to decouple these two. In the future I guess we will have func to upload to gcp, which might have a diff request body format and header content too.